package.json

```
@@ -76,5 +76,6 @@
     "ts-mockito": "^2.6.1",
     "ts-node": "^10.9.2",
     "typescript": "^5.6.2"
-  }
+  },
+  "packageManager": "pnpm@9.12.0+sha512.4abf725084d7bcbafbd728bfc7bee61f2f791f977fd87542b3579dcb23504d170d46337945e4c66485cd12d588a0c0e570ed9c477e7ccdd8507cf05f3f92eaca"
```

Whats this for?

---

This field is added automatically by corepack if not set - probably an accidental commit though.
It can be used to control the version of the package manager though, especially when corepack is used for managing package managers
```suggestion
  "packageManager": "pnpm@9.12.0"
```